backend/clubs/views.py

```diff
@@ -5505,6 +5547,13 @@ def transfer(self, request, *args, **kwargs):
         return Response({"detail": "Successfully transferred ownership of ticket"})
 
     def get_queryset(self):
+        if self.action == "partial_update":
+            officer_clubs = Membership.objects.filter(
+                person=self.request.user, role__lte=Membership.ROLE_OFFICER
+            ).values_list("club", flat=True)
+            return Ticket.objects.filter(event__club__in=officer_clubs).select_related(
+                "event__club"
+            )
         return Ticket.objects.filter(owner=self.request.user.id)
```

Comment:
A bit nasty, but I guess it gets the job done. We should revisit later.

Reply:
We'll need something similar for the QR code validation route, since that should also be accessible by club officers + ticket owner

Reply:
Should we do something similar to how perms work for `create_tickets`?